### PR TITLE
DEVDOCS-2297: Remove client id: update products-overview.md

### PR DESCRIPTION
**Pull request by traci707**
# [DEVDOCS-2297](https://jira.bigcommerce.com/browse/DEVDOCS-2297)

## What changed?
Client_id removed.
This PR will contain changes made to files under api-docs where client id has been removed.

### CODE REVIEW

File: docs/api-docs/apps/guide/apps-03-developing.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ Once you've exposed an app to the internet, you can register a draft app in [Dev
    * **Uninstall**: `https://4022ffe4.ngrok.io/uninstall`
 6.  Click **Update & Close**.
 
-7.  Click **View Client ID** to view the app's `client_id` and `client_secret`.
```
Comment (aglensmith) [inline docs/api-docs/apps/guide/apps-03-developing.md:85]: Are we sure `client_id` is going to be removed from here and will not be used for anything?

File: docs/api-docs/apps/guide/apps-10-buttons.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ First, embed an install button like the one below, at any web location from whic
 
 ![Install Button](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/apps-10-buttons-01.png "Install Button")
 
-Redirect anyone who presses your button to: `https://login.bigcommerce.com/app/{{CLIENT_ID}}/install`.
+Redirect anyone who presses your button to: `https://login.bigcommerce.com/app/install`.
```
Comment (aglensmith) [inline docs/api-docs/apps/guide/apps-10-buttons.md:26]: Are you sure this is correct?

File: docs/api-docs/customers/customer-login-api.md [hunk 3]
```diff
@@ -173,13 +171,12 @@ function getLoginUrl(customerId, storeHash, storeUrl, clientId, clientSecret) {
...
 };
  
-const clientId = “Your client id”;
 const clientSecret = “Your client secret”;
 const customerId = “You customer id”;
 const storeHash = “Your store hash”;
 const storeUrl = “Your store url”;
  
-const loginUrl = getLoginUrl(customerId, storeHash, storeUrl, clientId, clientSecret);
+const loginUrl = getLoginUrl(customerId, storeHash, storeUrl, clientSecret);
```
Comment (aglensmith) [inline docs/api-docs/customers/customer-login-api.md:179]: We can't just remove the clientId param here; we might need to re-write all the example code.

**Review by aglensmith — CHANGES_REQUESTED** [review]

File: docs/api-docs/apps/guide/apps-10-buttons.md (hunk 2 above)
Comment (traci707) [reply docs/api-docs/apps/guide/apps-10-buttons.md:26]: This is not correct. I emailed Nathan on Friday asking him about the exceptions to the rule. He just got back with me today and this url and requests stays as it is as well as the others on this page.

**Review by uwikaiddi — APPROVED** [review]

**Pull request merged**